CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [8.2.1](https://github.com/awslabs/fhir-works-on-aws-interface/compare/v8.1.1...v8.2.1) (2021-05-10)
```

Comment:
Should this be `8.2.0`?

Reply:
Out of curiosity, was this created using `standard-version` or by hand. If it's `standard-version` and it bumped it to `8.2.1` then nvm about this comment, `8.2.1` is probably correct.